<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackMainWindow</class>
 <widget class="QMainWindow" name="PlaybackMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlaybackMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="topToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBackShowFile"/>
   <addaction name="actionNewShowFile"/>
   <addaction name="actionOpenShowFile"/>
   <addaction name="actionSaveShowFile"/>
   <addaction name="actionSettingsShowFile"/>
  </widget>
  <action name="actionSaveShowFile">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenShowFile">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSettingsShowFile">
   <property name="icon">
    <iconset theme="document-properties"/>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Edit Settings</string>
   </property>
  </action>
  <action name="actionBackShowFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="go-previous"/>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Go Back</string>
   </property>
  </action>
  <action name="actionNewShowFile">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
